modules/ROOT/pages/implicit-configurations.adoc

```diff
@@ -0,0 +1,110 @@
+= Implicit Configurations
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Some configurations can also be classified as implicit. A configuration can be used implicitly when none of its parameters are required. For example, suppose a foo configuration object which has a required parameter bar:
+
+[source, java, linenums]
+----
+<example:foo-config name="myFoo" bar="required value" />
+
+<flow>
+	<example:operation config-ref="myFoo" />
+</flow>
+----
+
+Now suppose that the bar parameter were to turn optional instead:
+
+[source, java, linenums]
+----
+<example:foo-config name="myFoo" />
+
+<flow>
+	<example:operation config-ref="myFoo" />
+</flow>
+----
+
+In this case, the entire foo-config (and subsequent config-ref) are redundant as they have no real effect. In these cases, implicit configurations can be used:
+
+[source, java, linenums]
+----
+<flow>
+	<example:operation />
+</flow>
+----
+
+Because `foo-config` does not have required parameters, the SDK can create an implicit instance in which all parameters are automatically assigned to their default values. The implicit configuration is either static or dynamic depending on its parameter’s default values. If any of those default values is an expression, then the implicit configuration is dynamic.
```

Comment:
```suggestion
Because `my-config` does not have required parameters, the SDK can create an implicit instance in which all parameters are automatically assigned to their default values. The implicit configuration is either static or dynamic depending on its parameter’s default values. If any of those default values is an expression, then the implicit configuration is dynamic.
```